modules/objdetect/src/aruco/aruco_detector.cpp

```diff
@@ -847,15 +665,232 @@ struct ArucoDetector::ArucoDetectorImpl {
                       const RefineParameters& _refineParams): dictionary(_dictionary),
                       detectorParams(_detectorParams), refineParams(_refineParams) {}
 
-    float getAverageArucoPinSize(vector<Point2f> markerCorners) {
+    float getAverageModuleSize(vector<Point2f> markerCorners) {
         float averageArucoModuleSize = 0.f;
         int numPins = dictionary.markerSize + detectorParams.markerBorderBits * 2;
         for (size_t i = 0ull; i < markerCorners.size(); i++) {
             averageArucoModuleSize += sqrt(normL2Sqr<float>(Point2f(markerCorners[i] - markerCorners[(i+1ull)%markerCorners.size()])));
         }
```

Comment:
It's duplicate. Previous instance of the function may be reused.

Reply:
removed duplicate